### PR TITLE
changed styles of verification process icons

### PR DESCRIPTION
**Pull request by Mi-Lan**

### CODE REVIEW

**Comment by pimato** [comment]
This:
Bildschirmfoto 2020-11-17 um 19 10 41

Should be a hyperlink to the kleros curate market page

**Review by pimato — CHANGES_REQUESTED** [review]
see comments

**Comment by pimato** [comment]
Also, can we have the onHover only when the user is actually hovering above the text. Currently the hover effect shows also if I am far away from it:
IMG_4353

**Review by pimato — APPROVED** [review]

**Review by liaojikunwork — APPROVED** [review]

File: app/src/components/market/sections/market_verify/option/kleros_curation.tsx [hunk 1]
```diff
@@ -131,6 +164,10 @@ export const KlerosCuration: FC<Props> = (props: Props) => {
   let KlerosNotice
   let KlerosRightColumn
 
+  const urlSwitch = (url: string) => {
+    const win = window.open(url, '_blank')
+    win && win.focus()
+  }
```
Comment (KadenZipfel) [inline app/src/components/market/sections/market_verify/option/kleros_curation.tsx:170]: The problem with this approach is that [browser support is unknown](https://caniuse.com/?search=window.open). I think we're better off staying on the safe side and using what we know works for sure

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**